tests/utils.py

```diff
@@ -119,7 +119,10 @@ def area_difference(Rr1, Rr2, Zr1, Zr2, Rv1, Rv2, Zv1, Zv2):
             for poly1, poly2 in zip(poly_r1.flat, poly_r2.flat)
         ]
     ).reshape((Rr1.shape[2], Rr1.shape[0]))
-    area_rho = np.where(diff_rho > 0, diff_rho / intersect_rho, 0)
+    print(diff_rho, intersect_rho)
```

Comment:
why the print statement here? for when the test fails so you can more easily see what they were?
this is probably fine to keep as it doesn't affect the test at all and only gives more info when it fails

Reply:
this was an old debugging statement, its been removed